### PR TITLE
homepage created

### PR DESCRIPTION
**Pull request by cjinc121**
created homepage 
added navbar ,categories, banners and featured products
Screenshot (102)
Screenshot (103)

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+.banner-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  padding: 2rem;
+  width: 98%;
+  border: 2px solid black;
+  margin: 1%;
```
Comment (deepak29-git) [inline homepage.css:20]: use rem unit

**Review by deepak29-git — COMMENTED** [review]
Everything looks good

**Review by anjali1102 — APPROVED** [review]
UI and responsiveness are good
LGTM !

File: homepage.css (hunk 1 above)
Comment (cjinc121) [reply homepage.css:20]: not using rem since its a very small unit, it wont affect

**Review by deepak29-git — APPROVED** [review]

**Pull request merged**